<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>profil</class>
 <widget class="QDialog" name="profil">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Profil</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
background: rgb(168, 0, 0);
}</string>
  </property>
  <widget class="QCheckBox" name="numeCheckBox">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>44</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox{
border: 2px solid rgb(85, 0, 0);
border-radius:7px;
background: #D8B75A;
}</string>
   </property>
   <property name="text">
    <string>Éditer</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameLineEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border: 2px solid rgb(85, 0, 0);
border-radius:7px;
background: #D8B75A;
color: black;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>21</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
border: 2px solid rgb(85, 0, 0);
border-radius:7px;
background: #D8B75A;
font-size: 16px;
}</string>
   </property>
   <property name="text">
    <string>  Nom:</string>
   </property>
  </widget>
  <widget class="QLabel" name="prenumeLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>71</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
border: 2px solid rgb(85, 0, 0);
border-radius:7px;
background: #D8B75A;
font-size: 16px;
}</string>
   </property>
   <property name="text">
    <string>  Prénom:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="prenumeLineEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>80</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border: 2px solid rgb(85, 0, 0);
border-radius:7px;
background: #D8B75A;
color: black;
}</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="prenumeCheckBox">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>94</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox{
border: 2px solid rgb(85, 0, 0);
border-radius:7px;
background: #D8B75A;
}</string>
   </property>
   <property name="text">
    <string>Éditer</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="CNPCheckBox">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>144</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox{
border: 2px solid rgb(85, 0, 0);
border-radius:7px;
background: #D8B75A;
}</string>
   </property>
   <property name="text">
    <string>Éditer</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="CNPLineEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>130</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border: 2px solid rgb(85, 0, 0);
border-radius:7px;
background: #D8B75A;
color: black;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="CNPLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>121</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
border: 2px solid rgb(85, 0, 0);
border-radius:7px;
background: #D8B75A;
font-size: 16px;
}</string>
   </property>
   <property name="text">
    <string>  CNP:</string>
   </property>
  </widget>
  <widget class="QLabel" name="dataNasteriiLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>271</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
border: 2px solid rgb(85, 0, 0);
border-radius:7px;
background: #D8B75A;
font-size: 16px;
}</string>
   </property>
   <property name="text">
    <string>  Date de naissance:</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dataNasteriiEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>280</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QDateEdit{
border: 2px solid rgb(85, 0, 0);
border-radius:7px;
background: #D8B75A;
color: black;
}</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="dataNasteriiCheckBox">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>294</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox{
border: 2px solid rgb(85, 0, 0);
border-radius:7px;
background: #D8B75A;
}</string>
   </property>
   <property name="text">
    <string>Éditer</string>
   </property>
  </widget>
  <widget class="QLabel" name="adresaLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>221</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
border: 2px solid rgb(85, 0, 0);
border-radius:7px;
background: #D8B75A;
font-size: 16px;
}</string>
   </property>
   <property name="text">
    <string>  Adresse:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="adresaLineEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>230</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border: 2px solid rgb(85, 0, 0);
border-radius:7px;
background: #D8B75A;
color: black;
}</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="adresaCheckBox">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>244</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox{
border: 2px solid rgb(85, 0, 0);
border-radius:7px;
background: #D8B75A;
}</string>
   </property>
   <property name="text">
    <string>Éditer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="schimbaParolaButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>390</y>
     <width>251</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border: 2px solid rgb(85, 0, 0);
border-radius:7px;
background: #D8B75A;
font-size: 20px;
}</string>
   </property>
   <property name="text">
    <string>Changer le mot de passe</string>
   </property>
  </widget>
  <widget class="QPushButton" name="confirmaButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>390</y>
     <width>251</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border: 2px solid rgb(85, 0, 0);
border-radius:7px;
background: #D8B75A;
font-size: 20px;
}</string>
   </property>
   <property name="text">
    <string>Confirme les changements</string>
   </property>
  </widget>
  <widget class="QLabel" name="serieBuletinLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>171</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Calibri,sans-serif'; font-size:11pt;&quot;&gt;Le numéro de la carte d’identité nationale:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
border: 2px solid rgb(85, 0, 0);
border-radius:7px;
background: #D8B75A;
font-size: 16px;
}</string>
   </property>
   <property name="text">
    <string>  Le numéro de la...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="serieBuletinJudLineEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>180</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border: 2px solid rgb(85, 0, 0);
border-radius:7px;
background: #D8B75A;
color: black;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="serieBuletinCodLineEdit">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>180</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border: 2px solid rgb(85, 0, 0);
border-radius:7px;
background: #D8B75A;
color: black;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>180</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
color: white;
}</string>
   </property>
   <property name="text">
    <string> -</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="serieBuletinCheckBox">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>191</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox{
border: 2px solid rgb(85, 0, 0);
border-radius:7px;
background: #D8B75A;
}</string>
   </property>
   <property name="text">
    <string>Éditer</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="loculNasteriiCheckBox">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>340</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox{
border: 2px solid rgb(85, 0, 0);
border-radius:7px;
background: #D8B75A;
}</string>
   </property>
   <property name="text">
    <string>Éditer</string>
   </property>
  </widget>
  <widget class="QLabel" name="loculNasteriiLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>321</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
border: 2px solid rgb(85, 0, 0);
border-radius:7px;
background: #D8B75A;
font-size: 16px;
}</string>
   </property>
   <property name="text">
    <string>  Lieu de naissance:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="loculNasteriiLineEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>330</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border: 2px solid rgb(85, 0, 0);
border-radius:7px;
background: #D8B75A;
color: black;
}</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>nameLineEdit</tabstop>
  <tabstop>numeCheckBox</tabstop>
  <tabstop>prenumeLineEdit</tabstop>
  <tabstop>prenumeCheckBox</tabstop>
  <tabstop>CNPLineEdit</tabstop>
  <tabstop>CNPCheckBox</tabstop>
  <tabstop>serieBuletinJudLineEdit</tabstop>
  <tabstop>serieBuletinCheckBox</tabstop>
  <tabstop>adresaLineEdit</tabstop>
  <tabstop>adresaCheckBox</tabstop>
  <tabstop>dataNasteriiEdit</tabstop>
  <tabstop>dataNasteriiCheckBox</tabstop>
  <tabstop>loculNasteriiLineEdit</tabstop>
  <tabstop>loculNasteriiCheckBox</tabstop>
  <tabstop>schimbaParolaButton</tabstop>
  <tabstop>confirmaButton</tabstop>
  <tabstop>serieBuletinCodLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
